pvlib/shading.py

```diff
@@ -191,3 +191,117 @@ def sky_diffuse_passias(masking_angle):
        Available at https://www.nrel.gov/docs/fy18osti/67399.pdf
     """
     return 1 - cosd(masking_angle/2)**2
+
+
+def tracker_shaded_fraction(tracker_theta, gcr, projected_solar_zenith,
+                            cross_axis_slope=0):
+    r"""
+    Shade fraction (FS) for trackers with a common angle on an east-west slope.
```

Comment:
```suggestion
    Shade fraction for trackers with a common angle on an east-west slope.
```
The acronym is never used anywhere

Reply:
>Doesn't this function do the same as this existing function: `pvlib.bifacial.infinite_sheds._shaded_fraction()`

They differ in that the new function allows the array to be on a cross axis slope. If we adapt the infinite sheds model to allow arbitrary orientation, then we could reuse this function, but that work was delayed due to some questions about where the horizon should be and what the sky and ground view factors would include. Until those issues are resolve and that enhancement is implemented, this new shaded fraction function is more generalized.

Reply:
>The acronym is never used anywhere

It was the acronym that @williamhobbs used in #1689 and it's used here:

https://github.com/mikofski/pvlib-python/blob/dee9f12350dfcd49c372fe8a004fdfb44128b9f1/pvlib/shading.py#L269

and here:

https://github.com/mikofski/pvlib-python/blob/ffd9590a2c68e4030d498205711adf8ee3779106/pvlib/shading.py#L301

Reply:
I used "FS" for shaded fraction because Lorenzo 2011 used it, but I think it is confusing - should it be F<sub>s</sub>, it's easily confused with "First Solar" in this context, etc.